pkg/parquetconverter/converter.go

```diff
@@ -345,22 +364,31 @@ func (c *Converter) convertUser(ctx context.Context, logger log.Logger, ring rin
 		}
 
 		level.Info(logger).Log("msg", "converting block", "block", b.ULID.String(), "dir", bdir)
+		extraOpts := []convert.ConvertOption{
+			convert.WithSortBy(labels.MetricName),
+			convert.WithColDuration(time.Hour * 8),
+			convert.WithRowGroupSize(c.cfg.MaxRowsPerRowGroup),
+			convert.WithName(b.ULID.String()),
+		}
+
+		if c.cfg.FileBufferEnabled {
+			extraOpts = append(extraOpts, convert.WithColumnPageBuffers(parquet.NewFileBufferPool(bdir, "buffers.*")))
+		}
```

Comment:
Can we put those options outside of the for loop? Only `convert.WithName(b.ULID.String())` is the dynamic one